pages/community/[contractAddress]/index.tsx

```diff
@@ -25,6 +36,9 @@ export default function CommunityPage({ contractAddress }: CommunityPageProps) {
         chain: 'Ethereum',
       },
       forceRefresh: false,
+      tokenCommunityFirst: GRID_ITEM_PER_PAGE,
+      listOwnersFirst: LIST_ITEM_PER_PAGE,
+      onlyGalleryUsers: isFetchAllUsers,
```

Comment:
hmm i think `onlyGalleryUsers` and `isFetchAllUsers` are technically opposite statements! so maybe we could do:

```js
const fetchUniversalOwners = ART_GOBBLER_ADDRESS.includes(contractAddress);

...
onlyGalleryUsers: !fetchUniversalOwners
```

Reply:
Updated! Totally make sense